### PR TITLE
Minor Changes to output merge statements

### PR DESCRIPTION
**Pull request by bluisana**
expanded the blanknode identification by passing the entire nodelabel to the blanknode method. 

changed the nodetonodelabel method to output multiple labels as an array rather than a colon separated string.

### CODE REVIEW

File: src/jsonGraphToCypher.js [hunk 1]
```diff
@@ -110,11 +110,17 @@ const jsonGraphToCypher = async (graph, sourceGraphId) => {
       }
       typedProperties = `,  ${rps.join(', ')}`.trim();
     }
+
+    // testing to get to our prob nodes
```
Comment (nissimsan) [inline src/jsonGraphToCypher.js:114]: This needs to go

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: vp2.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+              "postalCode": "CB3 8NW",
+              "addressCountry": "UK"
+            }
+          },
+          "contactPoint": {
+            "type": ["ContactPoint"],
+            "place": {
+              "type": ["Place"],
+              "geo": {
+                "type": ["GeoCoordinates"]
```
Comment (nissimsan) [inline vp2.json:40]: These empty elements are quite weird.

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: src/jsonGraphToCypher.js [hunk 3]
```diff
@@ -110,11 +110,25 @@ const jsonGraphToCypher = async (graph, sourceGraphId) => {
...
       typedProperties = `,  ${rps.join(', ')}`.trim();
     }
+
     const nodeLinks = findNodeLink(node, graph.links);
     const nodeLabel = nodeToNodeLabel(node, nodeLinks);
     const typeNode = nodeLinks.length === 0;
-    const blanKNode = isBlankNode(node.id);
-    const nodeId = blanKNode ? '' : `{ id: "${node.id}" }`;
+
+    // check if the node or nodelabel have blank node identifiers in them
```
Comment (nissimsan) [inline src/jsonGraphToCypher.js:118]: 😍 Comments!!!

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: src/jsonGraphToCypher.js [hunk 4]
```diff
@@ -110,11 +110,25 @@ const jsonGraphToCypher = async (graph, sourceGraphId) => {
...
+    const blanKNodeID = isBlankNode(node.id);
+    const blanKNodeLabel = isBlankNode(nodeLabel);
+
+    // if either nodeid or the nodelable have blank node identifiers then this is a blank node
```
Comment (nissimsan) [inline src/jsonGraphToCypher.js:122]: 😍 Comments!!!

File: vp2.json (hunk 2 above)
Comment (bluisana) [reply vp2.json:40]: irrelevant file has been removed.

**Review by nissimsan — APPROVED** [review]
This looks solid

**Pull request merged**